packages/expect/src/matchers.ts

```diff
@@ -137,7 +137,26 @@ const matchers: MatchersObject = {
       secondArgument,
       secondArgumentColor: (arg: string) => arg,
     };
-    ensureNumbers(received, expected, matcherName, options);
```

Comment:
why is this changed?

Reply:
ah, https://github.com/facebook/jest/pull/8382#issuecomment-487415080. @pedrottimark do you still stand by that conclusion?

Reply:
If out of that wordy mess, you mean that `toBeCloseTo` should be limited to `number` type, yes I do

Reply:
that's exactly what I meant 😀 Should we have a caveat about that in the docs? Might be surprising that you cannot do `expect(1n).toBeCloseTo(1n)`

Reply:
Yes, you remind me what I had forgotten that this PR doesn’t have doc updates.

What do you think if I do that as a chore to be merged whenever you merge this?

Ha ha, that assertion is Jest contribution to *** JS equality confusion, because any integer (let alone bigint) comparison to self will fail because default precision is `0.01`

Reply:
Sorry I confused myself, it would pass, of course. The problem is although spec supports comparisons between number and bigint in the 4 matchers, computation of a range seems subtle and display of the difference in a failing report even more so.

Reply:
You can push doc updates to this PR, can't you?